ppm-domain/src/main/java/com/ctu/se/oda/model11/models/commands/requests/material/CreateMaterialCommandRequest.java

```diff
@@ -12,21 +13,18 @@
 @RequiredArgsConstructor
 @Builder
 public class CreateMaterialCommandRequest {
-    @NotBlank
     @Size(max = 250)
     private String materialName;
-    @NotNull
-    @Positive
-    private Long materialType;
-    public CreateMaterialCommandRequest(String materialName, Long materialType) {
+    private MaterialType materialTypeName;
```

Comment:
should change materialTypeName -> materialType

Reply:
Has been changed materialTypeName to materialType